### ISSUE TITLE
Webhook Error: URL references an IANA reserved block.

### ISSUE THREAD

**Issue by ikifar2012** [AzuraCast/AzuraCast#6350]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

AzuraCast • Rolling Release #69f4ed6 (2023-06-14 15:43) 

### What happened?

I am unable to send now playing updates out using webhooks,
When pushing test I get the following log output 
After updating it stopped working, I checked my Discord server where the webhook was active and the last message I have from it was May 31st 2023 but I was behind on updates and on rolling release so I can't tell exactly when this bug appeared

### Relevant log output

```shell
[2023-06-15T06:22:36.647852+00:00] AzuraCast.DEBUG: Writing local nowplaying text file... [] []
[2023-06-15T06:22:36.649432+00:00] AzuraCast.DEBUG: Writing static nowplaying text file... [] []
[2023-06-15T06:22:36.653173+00:00] AzuraCast.DEBUG: HTTP client POST call to http://localhost:6025/api produced response 200 [] []
[2023-06-15T06:22:36.667460+00:00] AzuraCast.ERROR: /var/azuracast/www/src/Webhook/Connector/AbstractConnector.php L129: URL references an IANA reserved block. {"exception":"[object] (RuntimeException(code: 0): URL references an IANA reserved block. at /var/azuracast/www/src/Webhook/Connector/AbstractConnector.php:129)"} []
```

**Comment by Jaadoo13** [comment]
Hello. I have a similar Problem. When i want to sent song Updates via Webhooks to my php-script, it get everytime a Timeout by curl. It seems, that curl requests cannot reach outside IPs. Doesnt matter, wich timeout im using for Curl. And ya, this Problem startet after Updating to the latest Rolling Version. Any Solution?
thx. Kinly, sven

``

**Comment by ikifar2012** [comment]
We need a toggle for the changes introduced in commit https://github.com/AzuraCast/AzuraCast/commit/cc67f8ccd80b6d079101d0cb2f759888da0d1733

**Comment by BusterNeece** [comment]
So first, an explanation for why we did things the "IANA reserved block" way: We received a security vulnerability report that basically said that a station operator (who doesn't own the AzuraCast server) could use generic webhooks to probe around the internal network visible to AzuraCast and see the full response bodies of any URL they entered. The easy and immediate solution to this was to prevent webhooks from dispatching to any IANA-reserved blocks, as those all correspond to "internal" URLs (think 192.168.x.x). Of course, this created a fair amount of problem in legitimate use-cases as well.

I've gone back and rethought how we handle this particular possible vulnerability and taken a different approach: now, if you're on a production installation and you don't have system-level administrator access, you can still test webhooks but will only see "Info"-level logs, not the "Debug"-level logs that contain raw HTTP response bodies. If you are either on a dev installation or have system-level administrator access, you'll automatically see debug information when testing those same webhooks. Thus, the regular function of the webhook is unimpeded and the only impacted section is the test debug output, which was what allowed exfiltration of internal data to begin with.

Same basic problem solved, just in a better way. Pushed up to Rolling and will be included in 0.19.1.

**Comment by ikifar2012** [comment]
Thank you!!!
